dev-support/test-patch.sh

```diff
@@ -2711,18 +2465,7 @@ function check_patch_javadoc
     fi
 
     personality_modules patch javadoc
-    case ${BUILDTOOL} in
-      maven)
-        modules_workers patch javadoc clean javadoc:javadoc
-      ;;
-      ant)
-        modules_workers patch javadoc clean javadoc
-      ;;
-      *)
-        yetus_error "ERROR: Unsupported build tool."
-        return 1
-      ;;
-    esac
+    ${BUILDTOOL}_modules_worker patch javadoc
```

Comment:
shellcheck: 18: note: Double quote to prevent globbing and word splitting. [SC2086]